models/Recording.js

```diff
@@ -121,10 +157,13 @@ module.exports = function(sequelize, DataTypes) {
       addAssociations: addAssociations,
       query: query,
       getOne: getOne,
+      deleteOne: deleteOne,
+      updateOne: updateOne,
       processingAttributes: processingAttributes,
       processingStates: processingStates,
       apiSettableFields: apiSettableFields,
       userGetAttributes: userGetAttributes,
+      apiUpdatableFields: apiUpdatableFields,
```

Comment:
there's probably no need to export this is there? It's only used here.

Really, I don't think any field names should be exported from the models.

Reply:
Yeah, probably not needed.

Reply:
Fixed